functions.js

```diff
@@ -32,17 +32,19 @@ const generateJourneySummaries = (journeys, originCode) => {
             const from = firstStation.name
             const to = lastStation.name
             const line = firstStation.line
-            const stationBreakdown = journey.map(station => station.name)
+            const time = journey.map(station => station.timeToNext).reduce((accumulator, currentValue) => accumulator + currentValue)
+            const stationBreakdown = journey.map(station => [station.name, station.timeToNext])
             summaries.push({
-                from: from, to: to, line: line, stations: stationBreakdown
+                from: from, to: to, line: line, time: time, stations: stationBreakdown
             })
         } else {
             const from = lastStation.name
             const to = firstStation.name
             const line = lastStation.line
-            const stationBreakdown = journey.map(station => station.name).reverse()
+            const time = journey.map(station => station.timeToPrev).reduce((accumulator, currentValue) => accumulator + currentValue)
+            const stationBreakdown = journey.map(station => [station.name, station.timeToPrev]).reverse()
             summaries.push({
-                from: from, to: to, line: line, stations: stationBreakdown
+                from: from, to: to, line: line, time: time, stations: stationBreakdown
             })
```

Comment:
Again, a bit of repetition here, consider making a function out of these

Reply:
agreed on not doing this